modules/team/versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.0"
+
+  required_providers {
+    github = {
+      source  = "integrations/github"
```

Comment:
Why not https://registry.terraform.io/providers/Flaconi/github/latest?

Reply:
This is not an use-case for that fork